src/odemis/acq/align/test/roi_autofocus_test.py

```diff
@@ -135,7 +140,7 @@ def test_estimate_autofocus_in_roi(self):
         """
         Tests time estimation of autofocus in roi
         """
-        n_focus_points = (3, 3)
-        min_time = n_focus_points[0] * n_focus_points[1] * estimateAutoFocusTime(self.ccd, None)
+        n_focus_points = 9
```

Comment:
The focus points vary based on the selected roi by the user. It can vary from 4 to 16 focus points.  To have fast execution of the test cases, I think 9 or less than that should be sufficient for the testing.